File: Kotlin/1020.kt
import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`);
    val edad = input.nextInt();
    val ano = edad/365;
    val mes = (edad%365)/30;
    val dia = (edad%365)%30;
    println("$ano ano(s)\n$mes mes(es)\n$dia dia(s)");
    
}

File: Javascript/1020.js
var input = `400`;
var lines = input.split("\n");
let edad = lines[0];
let ano = parseInt(edad / 365);
let mes = parseInt((edad % 365) / 30);
let dia = parseInt((edad % 365) % 30);
console.log(`${ano} ano(s)`);
console.log(`${mes} mes(es)`);
console.log(`${dia} dia(s)`);
